datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/metadata.py

```diff
@@ -289,24 +289,24 @@ def metadata(check):
                 elif len(row['description']) > MAX_DESCRIPTION_LENGTH:
                     errors = True
                     echo_failure(
-                        '{}: `{}` exceeds the max length: {} for descriptions.'.format(
-                            current_check, row['metric_name'], MAX_DESCRIPTION_LENGTH
+                        '{}:{} `{}` exceeds the max length: {} for descriptions.'.format(
+                            current_check, actual_line, row['metric_name'], MAX_DESCRIPTION_LENGTH
                         )
                     )
 
         for header, count in iteritems(empty_count):
             errors = True
-            echo_failure('{}: {} is empty in {} rows.'.format(current_check, header, count))
+            echo_failure('{}:{} {} is empty in {} rows.'.format(current_check, actual_line, header, count))
 
         for header, count in iteritems(empty_warning_count):
-            echo_warning('{}: {} is empty in {} rows.'.format(current_check, header, count))
+            echo_warning('{}:{} {} is empty in {} rows.'.format(current_check, actual_line, header, count))
 
         for prefix, count in iteritems(metric_prefix_count):
             # Don't spam this warning when we're validating everything
             if check:
                 echo_warning(
-                    '{}: `{}` appears {} time(s) and does not match metric_prefix '
-                    'defined in the manifest.'.format(current_check, prefix, count)
+                    '{}:{} `{}` appears {} time(s) and does not match metric_prefix '
+                    'defined in the manifest.'.format(current_check, actual_line, prefix, count)
                 )
```

Comment:
Is the `actual_line` valid for the last 3 cases ?  Seems we are outside of `for row in reader:`

- `for header, count in iteritems(empty_count):`
- `for header, count in iteritems(empty_warning_count):`
- `for prefix, count in iteritems(metric_prefix_count):`

Reply:
Good catch, I did it blindly